docs/cookbook/extract_from_pdf.md

```diff
@@ -0,0 +1,133 @@
+# Extracting from PDF: Guide to Extract Pages and Content from PDF Files
+
+This recipe demonstrates how to leverage Large Language Models (LLMs) -- specifically the OpenAI API -- to extract pages and content from PDF files. We'll cover single PDF document as well as multiple PDF files and also use OCR to extract text from scanned documents.
+
+??? info "Key Concepts"
+
+    - [Pydantic](ADD LINK)
```

Comment:
I think we can drop pydantic since we have response models, which will cover pydantic.

We should also include links in order of the learn section and include prompts here as well

Reply:
done